public/apps/configuration/panels/get-started.tsx

```diff
@@ -158,7 +160,17 @@ const setOfSteps = [
   },
 ];
 
+export function GetClusterDescription(dataSourceEnabled: boolean, cluster: Cluster) {
+  if (dataSourceEnabled) {
+    return `for ${cluster.label || 'Local cluster'}`;
+  }
+  return '';
+}
+
 export function GetStarted(props: AppDependencies) {
+  const dataSourceEnabled = !!props.securityPluginStartDeps.dataSource?.dataSourceEnabled;
+  const [dataSource, setDataSource] = useState<Cluster>({ id: '', label: '' });
```

Comment:
can you add a test that this useState exhibits correct behavior?

Reply:
Since the component that consumes this function lives in core, I found it hard to write a unit test for it. I believe the cypress test case that I wrote sufficiently covers this, since it is testing that the message in the toast (which relies on the dataSource set by the setDataSource function) is correctly being displayed after selecting the cluster in the dropdown. Do you think that's enough?